<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
	background-color: rgb(255, 255, 255);
    border: 1px solid #CCCCCC; /* Kolor i styl ramki */
    background: white; /* Kolor tła */
}

QTabBar::tab {
    background: #F0F0F0; /* Kolor tła nieaktywnej zakładki */
    border: 1px solid #CCCCCC; /* Ramka zakładek */
    border-bottom: none; /* Usunięcie dolnej krawędzi */
    padding: 5px 15px; /* Wewnętrzne marginesy */
    color: black; /* Kolor tekstu */
}

QTabBar::tab:selected {
    background: #FFFFFF; /* Kolor tła aktywnej zakładki */
    color: black; /* Kolor tekstu aktywnej zakładki */
    border: 1px solid #007BFF; /* Podkreślenie wybranej zakładki */
}

QTabBar::tab:hover {
    background: #E6E6E6; /* Kolor zakładki podczas najechania myszką */
}
QPushButton {
    background-color: #007BFF; /* Kolor tła */
    color: white; /* Kolor tekstu */
    border-radius: 5px; /* Zaokrąglenie krawędzi */
    padding: 5px 10px; /* Marginesy wewnętrzne */
    font-size: 14px;
}

QPushButton:hover {
    background-color: #0056b3; /* Kolor podczas najechania myszką */
}

QPushButton:pressed {
    background-color: #003f7f; /* Kolor podczas kliknięcia */
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabSign">
       <attribute name="title">
        <string>Sign In</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>512</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="labelLogin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelEmail_3">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EMAIL:   </string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditEmail_Sign">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Enter Your Email</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelEmail_4">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Password:   </string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditPassword_Sign">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>*********</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="Line" name="line_2">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>12</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="pushButtonLogin">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxRemember">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Remember Me</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="labelError_Log">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRegister">
       <attribute name="title">
        <string>Register</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>512</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="labelRegister">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelFirstName_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>First Name:   </string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditFirstName_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Enter Your First Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelEmail_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EMAIL:   </string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditEmail_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Enter Your Email</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelPassword_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Password:   </string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditPassword_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>*********</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>12</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBoxAgreement_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>I agree to all the statements.
Terms of Service </string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButtonImage">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Insert Profile Image</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="pushButtonRegister">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelError_Reg">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="labelConn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;------- GET CONNECTED WITH ------- &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonTwitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGoogle">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFacebook">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelWelcomePage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#aaffff;&quot;&gt;Register Yourself &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#aaffff;&quot;&gt;We are happy To Welcome You&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc2.qrc"/>
 </resources>
 <connections/>
</ui>
